CONTRIBUTING.md

```diff
@@ -2,45 +2,47 @@
 
 ## General rules
 
-1. Keep it simple.  
-    We don't need 8 miles of install script.  
+1. Keep it simple.
+    We don't need 8 miles of install script.
 
-2. Keep it Small.  
-    Only use what is absolutely needed.  
+2. Keep it Small.
+    Only use what is absolutely needed.
 
-3. Try to stay in the stock containers.  
+3. Try to stay in the stock containers.
     If you need something in a container, open a PR in [my yolks repo](https://github.com/parkervcp/yolks) where I can review and pull it up to the main repo. In addition, there is a larger quantity of [images here](https://github.com/parkervcp/images) for you to use that have not been migrated to Yolks yet.
 
-4. Don't be afraid to submit PR's to the egg repo.  
+4. Don't be afraid to submit PR's to the egg repo.
     I don't bite. I will work with you on the egg and the required things to run it.
 
-## Step 1.
-#### Be aware of the pterodactyl install process.
+## Step 1
+
+### Be aware of the pterodactyl install process
 
 The Pterodactyl install process is fairly simple once you know.
-    
+
+```md
```

Comment:
The spacing here can make for an easier reading experience.

Reply:
Yea I agree